ci-operator/jobs/openshift/verification-tests/openshift-verification-tests-master-postsubmits.yaml

```diff
@@ -0,0 +1,32 @@
+postsubmits:
+  openshift/verification-tests:
+  - agent: kubernetes
+    branches:
+    - ^master$
+    decorate: true
+    name: branch-ci-openshift-verification-tests-master-images
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --promote
```

Comment:
You'll want a promotion spec in the ci-op config so this doesn't go to the openshift release

Reply:
It will go to bushslicer is in ci namespace